test/TestExtensions/GenerateLicenseList/Program.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+
+namespace GenerateLicenseList
+{
+    class Program
+    {
+        static int Main(string[] args)
+        {
+            if (args.Count() != 3)
+            {
+                Console.WriteLine("This tool expects 3 arguments: licenses.json, exceptions.json, targetFileLocation");
+                return -1;
```

Comment:
I would add a note that this tool should be run using `UpdateNuGetLicenseSPDXList.ps1`